### ISSUE TITLE
Most config options don't work in `config.exs`?

### ISSUE THREAD

**Issue by nickjj** [mgwidmann/scrivener_html#74]
Hi,

If I set the following in my `config.exs` file, only the `view_style` takes effect:

```ex
config :scrivener_html,
  routes_helper: MyApp.Router.Helpers,
  view_style: :bootstrap_v4,
  distance: 5, next: "Next", previous: "Previous",
  first: true, last: true
```

I would have expected the other config options to be applied as well.

I wanted to do this because I have pagination on 4 different resources, and I didn't want to have to customize the next, previous, etc. for each one individually but currently that seems to be the only way to do it.

Also on a side note, first / last never show up, even if I set them directly on `pagination_links` in my eex template.

I'm running 1.7.1 (the latest release at the time of this issue).

**Comment by mgwidmann** [comment]
Yep, looks like we just pull the application environment config for those values
https://github.com/mgwidmann/scrivener_html/blob/master/lib/scrivener/html.ex#L98

I would accept a PR to make this addition.

**Comment by shulhi** [comment]
@mgwidmann I can definitely help with this once you have the time to review #86 and then I can work on top of that PR.

**Comment by kurisusan** [comment]
I have a little concern about scrivener config.
How should it be done for umbrella apps ?
Say I want to use bootstrap as viewstyle in one app and materialize_css in another...

can't we do something like:

```
config :admin, :scrivener_html,
  routes_helper: AdminWeb.Router.Helpers,
  view_style: :bootstrap_v4
```

and

```
config public, :scrivener_html,
  routes_helper: PublicWeb.Router.Helpers,
  view_style: :materialize
```

**Comment by mgwidmann** [comment]
Ping, going to close this soon if no one wants this feature.

**Comment by nickjj** [comment]
I think people do want it. I know I still do.
